data/json/items/tools.json

```diff
@@ -1373,7 +1373,7 @@
     "type": "TOOL",
     "name": "cell phone",
     "name_plural": "cell phones",
-    "description": "This is a typical smartphone that features an LED for use as a flash.  Using this cellphone will turn it on and provide light, assuming it is charged with batteries.",
+    "description": "This is a typical smartphone that features an LED for use as a flash and a clock and alarm app.  Using this cellphone will turn it on and provide light, assuming it is charged with batteries.",
```

Comment:
I think the grammar here is slightly confusing. The sentence you changed makes it seem as if the description is saying that the LED works as a clock and alarm app, as well as for flash. Here's what I recommend:

`"description": "This is a typical smartphone that features an LED for use as a flash.  Using this cellphone will turn it on and provide light, assuming it is charged with batteries. Also it has a clock app that includes an alarm.",`